board/piksiv3/rootfs-overlay/etc/init.d/S85piksi_firmware

```diff
@@ -2,10 +2,17 @@
 
 name="piksi_firmware"
 
+source /etc/init.d/common.sh
+
+log_tag=$name # required by logging.sh
+source /etc/init.d/logging.sh
+
+setup_loggers
+
 settings_ini=/persistent/config.ini
 
-uart_wait_count_limit=20 # wait 2s
-uart_check_interval=0.1
+uart_wait_count_limit=200 # wait 200s
```

Comment:
Let's put the interval back to 0.1 and the count to 2000, this way in the normal case we won't be adding a second to the boot time if we happen to miss the startup of the adapter.